core/shared/src/test/scala/cats/parse/LocationMapTest.scala

```diff
@@ -226,7 +226,7 @@ class LocationMapTest extends munit.ScalaCheckSuite {
         val lc = lm.toLineCol(offset)
 
```

Comment:
actually while we are at it can we add: `assertEquals(c.offset, offset)` to make sure we pass through the offset?